### ISSUE TITLE
key block from a WD with JMS528S chip

### ISSUE THREAD

**Issue by themaddoctor** [andlabs/reallymine#25]
00000000  57 44 76 31 b3 db 00 00  00 b8 bf d1 01 00 00 00  |WDv1³Û...¸¿Ñ....|
00000010  03 00 00 00 00 00 f0 00  00 00 00 00 00 00 00 00  |......ð.........|
00000020  01 00 00 00 00 00 46 50  00 00 00 00 00 00 00 00  |......FP........|
00000030  00 02 ff 00 00 00 00 00  00 00 00 00 00 00 00 00  |..ÿ.............|
00000040  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
00000050  20 00 3a 6a 00 00 00 01  00 00 00 00 57 44 76 31  | .:j........WDv1|
00000060  09 f8 45 57 df 43 28 50  2c 9e 4c 92 a0 93 b1 ed  |.øEWßC(P,.L. .±í|
00000070  1c 7e a7 1a 2a a5 8f 58  f5 06 c1 b5 6b 26 e7 18  |.~§.*¥.Xõ.Áµk&ç.|
00000080  5f d8 6e 2d 42 92 fe 5b  06 bc 30 b4 65 0f 87 b6  |_Øn-B.þ[.¼0´e..¶|
00000090  0c 17 d5 a6 34 5e a9 97  b2 3b bd 5c 24 82 57 e2  |..Õ¦4^©.²;½\$.Wâ|
000000a0  bb 34 a8 88 de 03 86 d9  bd 99 70 bb 61 a7 35 d9  |»4¨.Þ..Ù½.p»a§5Ù|
000000b0  92 d7 0a b4 ad 5a 94 dd  2e 8b 60 7f 22 df b8 32  |.×.´­Z.Ý..`."ß¸2|
000000c0  bf 6c f9 2c 38 65 47 90  59 2d b2 14 b8 07 bf 1b  |¿lù,8eG.Y-².¸.¿.|
000000d0  d1 0a 4b fb ab 90 a6 bb  f1 50 f2 4d 09 a7 f5 4b  |Ñ.Kû«.¦»ñPòM.§õK|
000000e0  d5 57 6e e4 64 46 0d 6b  75 42 9d 3a 91 6d 1d 61  |ÕWnädF.kuB.:.m.a|
000000f0  71 f7 75 13 d6 b4 d3 2a  4e 3c c3 40 eb 8f 56 8d  |q÷u.Ö´Ó*N<Ã@ë.V.|
00000100  c6 31 46 6c 7d 59 fb 8a  f9 84 02 96 02 e7 1b e5  |Æ1Fl}Yû.ù....ç.å|
00000110  2f cd b0 13 6c 88 3f f9  1b 3f 3a b8 51 f5 20 57  |/Í°.l.?ù.?:¸Qõ W|
00000120  c2 4a 55 db dd 13 28 aa  fa 4d 95 21 66 0c 13 48  |ÂJUÛÝ.(ªúM.!f..H|
00000130  8d 11 16 36 00 a2 60 ea  f4 43 35 a7 92 d9 4c 13  |...6.¢`êôC5§.ÙL.|
00000140  36 d9 68 8b ae e5 cc fa  8b a9 d8 b8 52 8d c3 1c  |6Ùh.®åÌú.©Ø¸R.Ã.|
00000150  6e d1 36 3d 51 43 2f 95  ee 6b eb 21 ea d8 45 a6  |nÑ6=QC/.îkë!êØE¦|
00000160  c2 09 55 bd b3 bd 0c b7  27 32 17 bb 23 e8 7d 79  |Â.U½³½.·'2.»#è}y|
00000170  7f 44 0f 2d 7f 22 99 22  d4 4f 02 1d 45 d3 49 25  |.D.-."."ÔO..EÓI%|
00000180  c0 d0 cf 3f c8 07 43 eb  12 28 d1 df ef ff 74 eb  |ÀÐÏ?È.Cë.(Ñßïÿtë|
00000190  9f a8 5a 1f 22 96 af 5a  57 56 b2 54 87 08 c5 b6  |.¨Z.".¯ZWV²T..Å¶|
000001a0  cf 1f 09 f0 f8 ed b1 d0  c0 a3 c9 26 b3 13 fc 47  |Ï..ðøí±ÐÀ£É&³.üG|
000001b0  80 e5 54 83 67 78 85 38  e2 57 92 aa 6d a1 3b bb  |.åT.gx.8âW.ªm¡;»|
000001c0  fb a9 fe 7e 55 ad 91 b3  0b 4f 08 9f 5f ec da cb  |û©þ~U­.³.O.._ìÚË|
000001d0  05 06 d6 64 0f 52 a8 5d  fc cc da fa 3e 9e 12 a8  |..Öd.R¨]üÌÚú>..¨|
000001e0  6f 94 59 43 62 43 50 b5  9a 92 bc 7a c7 01 5a 42  |o.YCbCPµ..¼zÇ.ZB|
000001f0  53 48 6e f6 e7 bb c8 7c  dc 0e 2c d1 bc 77 d1 40  |SHnöç»È|Ü.,Ñ¼wÑ@|

no password was set, so the extracted key is
3baf6055cc76eb00e612a392031f240a06575f756d8beb77f27a3e298b015028

but to use it, it must be reversed:
2850018b293e7af277eb8b6d755f57060a241f0392a312e600eb76cc5560af3b

**Comment by andlabs** [comment]
Could you provide this as a zip file?

**Comment by themaddoctor** [comment]
I tried, but
"We don’t support that file type. Try again with a PNG, GIF, JPG, DOCX, PPTX, XLSX, TXT, PDF, or ZIP. "

**Comment by themaddoctor** [comment]
None of my .zip files are acceptable, it seems.

Is this easier to use?

57447631b3db000000b8bfd101000000030000000000f0000000000000000000
010000000000465000000000000000000002ff00000000000000000000000000
0000000000000000000000000000000020003a6a000000010000000057447631
09f84557df4328502c9e4c92a093b1ed1c7ea71a2aa58f58f506c1b56b26e718
5fd86e2d4292fe5b06bc30b4650f87b60c17d5a6345ea997b23bbd5c248257e2
bb34a888de0386d9bd9970bb61a735d992d70ab4ad5a94dd2e8b607f22dfb832
bf6cf92c38654790592db214b807bf1bd10a4bfbab90a6bbf150f24d09a7f54b
d5576ee464460d6b75429d3a916d1d6171f77513d6b4d32a4e3cc340eb8f568d
c631466c7d59fb8af984029602e71be52fcdb0136c883ff91b3f3ab851f52057
c24a55dbdd1328aafa4d9521660c13488d11163600a260eaf44335a792d94c13
36d9688baee5ccfa8ba9d8b8528dc31c6ed1363d51432f95ee6beb21ead845a6
c20955bdb3bd0cb7273217bb23e87d797f440f2d7f229922d44f021d45d34925
c0d0cf3fc80743eb1228d1dfefff74eb9fa85a1f2296af5a5756b2548708c5b6
cf1f09f0f8edb1d0c0a3c926b313fc4780e5548367788538e25792aa6da13bbb
fba9fe7e55ad91b30b4f089f5fecdacb0506d6640f52a85dfcccdafa3e9e12a8
6f945943624350b59a92bc7ac7015a4253486ef6e7bbc87cdc0e2cd1bc77d140

**Comment by andlabs** [comment]
That'll do; thanks.

**Comment by themaddoctor** [comment]
Do you have a keyblock and start.bin for an Initio disk? I'd like to get that working, too.

**Comment by andlabs** [comment]
reallymine already handles Initio and some JMicron chip I forget which. Check inside there.

**Comment by themaddoctor** [comment]
> reallymine already handles Initio and some JMicron chip I forget which. Check inside there.
Right, but I want to try to get it working as a mounted drive in linux, rather than as a disk dump.
